modules/afprog/afprog.c

```diff
@@ -91,10 +91,72 @@ _exec_program(const gchar *cmdline)
   execl("/bin/sh", "/bin/sh", "-c", cmdline, NULL);
 }
 
+static void
+_closing_all_fd_expect(int ignore[], int ignore_len)
+{
+  for (int i = 10000; i > 5; --i)
```

Comment:
getrlimit(RLIMIT_NOFILE)

Reply:
thanks, I was just looking for this right now.